users/config.rst

```diff
@@ -980,6 +1173,75 @@ setLowPriority
     disable this behavior, for example to control process priority yourself
     as part of launching Syncthing, set this option to ``false``.
 
+.. option:: options.maxFolderConcurrency
+
+    This option controls how many folders may concurrently be in I/O-intensive
+    operations such as syncing or scanning.  The mechanism is described in
+    detail in a :doc:`separate chapter </advanced/option-max-concurrency>`.
+
+.. option:: options.crashReportingURL
+
+    Server URL where :doc:`automatic crash reports <crashrep>` will be sent if
+    enabled.
+
+.. option:: options.crashReportingEnabled
+
+    Switch to opt out from the :doc:`automatic crash reporting <crashrep>`
+    feature to avoid Syncthing "phoning home" on serious troubles.  Defaults to
+    ``true``, to help the developers troubleshoot.
+
+.. option:: options.databaseTuning
+
+    Controls how Syncthing uses the backend key-value database that stores the
+    index data and other persistent data it needs.  The available options and
+    implications are explained in a :doc:`separate chapter
+    </advanced/option-database-tuning>`.
+
+.. option:: options.maxConcurrentIncomingRequestKiB
+
+    This limits how many bytes we have "in the air" in the form of response data
+    being read and processed.
```

Comment:
I think this one needs to be explained more, in more "human language", if possible. Right now, I can't understand it, let alone any even less technical person.

Reply:
Me neither.  I'm living off what I could find as my understanding of Syncthing's code is also limited.  Whoever has time / energy to improve it, please do so.  But let's keep that in a separate PR.  My goal was to have completeness and not introduce additional TODO boxes.